src/main/java/org/apache/commons/net/ftp/parser/UnixFTPEntryParser.java

```diff
@@ -110,7 +110,7 @@ public class UnixFTPEntryParser extends ConfigurableFTPFileEntryParserImpl {
     /**
      * The default constructor for a UnixFTPEntryParser object.
      *
-     * @throws IllegalArgumentException Thrown if the regular expression is unparseable. Should not be seen under normal conditions. It it is seen, this is a
+     * @throws IllegalArgumentException Thrown if the regular expression is unparseable. Should not be seen under normal conditions. If it is seen, this is a
```

Comment:
Might this be better: "If it is seen" -> "If it is thrown"

Reply:
Would apply for many other occurrences as well. Could be a separate PR, wdyt?

Reply:
Up to you. When you are done with edits here, I will review the PR again (hopefully for the last time).

Reply:
Hm, then if I was confused by what "it" refers to, can we replace the "it" with the actual subject?

Reply:
I replaced "it" with "this exception" and left "seen" as is.